docs/_webCode/js/downloads.js

```diff
@@ -22,10 +21,7 @@ document$.subscribe(() => {
     }
   };
 
-  //This script might exist on every page due to instant loading but should only be executed on the downloads page
-  if (document.getElementById("download-all-release-build") !== null) {
```

Comment:
Why remove this check?

Reply:
I could not see, that the script is actually called for any other page. But may i did not looked up good enough

Reply:
I checked once more in the sources of the browser, and the script is not even in the sources on every other page.